src/main/webapp/src/repositories/user.repository.ts

```diff
@@ -0,0 +1,48 @@
+import {Injectable} from '@angular/core';
+import {User} from './model/user';
+
+const USERS = 'users';
+
+@Injectable()
+export class UserRepository {
+  /**
+   * Adding signed in user to storage
+   */
+  add(profile: gapi.auth2.BasicProfile) {
+    const users = this.getAll();
+
+    // Determine whether signed user id is already on local storage
+    let foundIndex = -1;
+    for (let i = 0; i < users.length; i++) {
+      if (users[i].id === profile.getId()) {
+        foundIndex = i;
+        break;
+      }
+    }
+    // Remove user at found index
+    if (foundIndex >= 0) {
+      users.splice(foundIndex, 1);
+    }
+
+    // Add and save signed user to storage
+    users.push(User.fromBasicProfile(profile));
+    this.save(users);
```

Comment:
Initially it was just made to help me as I was trying to figure out the authentication for the google calendar API (initially I thought I would need some info) but I have now removed the user repository since it is not being used in any way. Thank you for pointing it out!